ui.content/src/main/content/META-INF/vault/filter.xml

```diff
@@ -87,15 +88,21 @@
     <filter root="/oak:index/rep:policy"/>
     <filter root="/rep:policy"/>
     <filter root="/var/acs-commons">
-        <include pattern="/var/acs-commons/rep:policy"/>
+        <include pattern="/var/acs-commons/httpcache"/>
+        <include pattern="/var/acs-commons/httpcache/root"/>
+        <include pattern="/var/acs-commons/httpcache/rep:policy(/.*)?"/>
         <include pattern="/var/acs-commons/jcr:content"/>
+        <include pattern="/var/acs-commons/mcp/rep:policy(/.*)?"/>
```

Comment:
This is not being included - may need to add a filter for /var/acs-commons/mcp

Reply:
Yes, correct. I am gonna include the parent as well and will try to add a validation check for this.

Reply:
Resolved in https://github.com/Adobe-Consulting-Services/acs-aem-commons/pull/2045/commits/bc141861b764b2dff80ff9c35c51523e32a0973d